src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -228,8 +233,9 @@ const MintTokenMotion = ({
       </div>
     ),
   };
-  const motionStyles = MOTION_TAG_MAP[motionState || MotionState.Invalid];
 
+  const motionState = motionStatusData?.motionStatus || MotionState.Invalid;
```

Comment:
Because of this line, we're seeing for a short while invalid tag when data is loading. I think we need to add loading state to tag component

Reply:
Maybe not show it at all until we a "proper" state for it ?

Reply:
Yes, it make even more sense